x-pack/plugins/rule_registry/common/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const BASE_RAC_ALERTS_API_PATH = '/api/rac/alerts';
```

Comment:
**observation**: routes starting with `/api/` are intended for eventual public consumption. If we don't want to document these APIs for public use, then we should change this to something like `/internal/rac/alerts`.

Is `rac` something that is going to make sense once the "RAC" initiative is complete? Is this a proper domain that will exist within Kibana?

Reply:
Thanks so much for pointing this out. There's some ongoing conversations about this as there is an existing public signals API. So this API will need to be external. But for now renamed to be internal while details get hashed out.

https://github.com/elastic/kibana/pull/100705/commits/8d320376a54f21ec5f104669084a1c501f060e60#diff-b65106bd55912b5374ba12f2ce4e7c202df444f0210bdab8eda9245c70e179bcR8